source/administration/changelog.md

```diff
@@ -2,6 +2,210 @@
+- There is now a confirmation dialog before deleting a custom emoji
+
+#### Search
+- File attachments thumbnails are now shown in search results
+- Flagged posts from other teams are no longer displayed 
+
+#### Channels
+- Favorite channels are now sorted alphabetically regardless of channel type
+- Town Square now has a default channel purpose
+- User added to a group message are now removed from the Direct Messages search list
+- "Private Groups" have been renamed to "Private Channels"
+
+#### Mobile
+- Executing a search now closes the keyboard and removes the keyboard focus from the text box
+
+#### Integrations
+- The integrations confirmation page can now be dismissed with the ENTER key
+
+#### Notifications
+- Users can no longer configure email notification settings if the notifications are disabled for the system
+
+#### Onboarding
+- Existing users on the server can now easily be added to a team via the Main Menu
+
+#### Enterprise Edition
+- Policy controls for restricting permissions to add and remove members from private channels
+- Added the ability to read the license file from the disk
+
+### Bug Fixes
+- Fixed line wrapping of the timestamp in Account Settings > Security > View Access History 
+- Fixed an inconsistent error message when creating a channel with a display name of two or less characters
+- Removed the duplicate "Back" button on the Team Creation page
+- The AltGR key no longer triggers keyboard shortcuts
+- Saving a team name without making changes no longer throws an error message
+- Group messages are now sorted alphabetically with direct messages
+- The "Create Channel" button will no only appear in the "More Channels" modal when the user has the permission to create channels
+- The Town Square channel menu no longer has redundant dividers with certain combinations of System Console > Policy settings
+- Fixed an issue where some conversations would not trigger the channel to appear unread in the left-hand sidebar
+- Fixed an issue where usernames sometimes did not appear when hovering over reactions
+- Fixed missing search results when using the "in:" or "from:" modifiers
+- Fixed an issue where link previews would sometimes cause a horizontal scroll bar to appear
+- iOS code blocks no longer wrap to the next line
+
+### Compatibility  
+
+#### Breaking changes:
+- The **System Console > Configuration > [Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url)** field is now mandatory. Please set the Site URL in the System Console or in the `gitlab.rb` file if you are using GitLab Mattermost.
+- Server logs will now be written to the `mattermost.log` file located at the path specified in **System Console > Logging > [File Log Directory](https://docs.mattermost.com/administration/config-settings.html#file-log-directory)**. Please set the file path in the System Console or in the `gitlab.rb` file if you are using GitLab Mattermost.
+
```

Comment:
We should make it clear that a filename is not valid anymore... the value must be a directory path.